ldms/src/core/ldms.c

```diff
@@ -1243,6 +1261,37 @@ int ldms_set_perm_set(ldms_set_t s, mode_t perm)
         return 0;
 }
 
+void ldms_set_default_authz(uid_t *uid, gid_t *gid, mode_t *perm, int set_flags)
+{
+	if (set_flags == 0) {
```

Comment:
Let explicitly compare `set_flags == DEFAULT_AUTHZ_READONLY`. This would make the code easier to read.

Reply:
Sure.